code/game/turfs/closed/minerals.dm

```diff
@@ -598,11 +598,13 @@
 			G.icon_state = "Gibtonite ore 2"
 
 	var/flags = NONE
-	if(defer_change)
+	var/old_type = type
+	if(defer_change) // TODO: make the defer change var a var for any changeturf flag
 		flags = CHANGETURF_DEFER_CHANGE
-	ScrapeAway(null, flags)
-	addtimer(CALLBACK(src, .proc/AfterChange), 1, TIMER_UNIQUE)
-
+	var/turf/open/mined = ScrapeAway(null, flags)
+	addtimer(CALLBACK(src, .proc/AfterChange, flags, old_type), 1, TIMER_UNIQUE)
+	playsound(src, 'sound/effects/break_stone.ogg', 50, TRUE) //beautiful destruction
+	mined.update_visuals()
```

Comment:
I'm honestly not sure what the purpose of the update_visuals call wass, but I'm not going to question it.
The break sound prob? shouldn't play here, since it's only triggered by gibtonite being hit by a scanner. IDK

Reply:
This codebase is so cursed. I'll yank the sound out then.

Gibtonite has the same baseturfs as /closed/mineral so I figured I'd copycat the update_visuals since technically the `var/turf/open/mined` is the same as the parent version of this.

Reply:
aight that sounds good to me